app/helpers/lightweight_activity_helper.rb

```diff
@@ -59,6 +59,15 @@ def complete_badge_for(activity)
     end
   end
 
+  def activity_player_url(activity, mode="")
+    activity_api_url = "#{api_v1_activity_url(activity.id)}.json"
+    return  "#{ENV['ACTIVITY_PLAYER_URL']}/?activity=#{CGI.escape(activity_api_url)}&preview&mode=#{mode}"
```

Comment:
It'd be safer to use a URL utility here instead of just combining strings. Perhaps at some point the ACTIVITY_PLAYER_URL will have have its own parameters in it which would then break the code above.

Reply:
Sorry, I should've remembered that from your previous suggestion about using a URL utility elsewhere. Fixed.